specifyweb/frontend/js_src/lib/components/Statistics/definitions.ts

```diff
@@ -0,0 +1,26 @@
+import type { State } from 'typesafe-reducer';
```

Comment:
Could you rename this file to `types.ts`? Files that only include typescript types are usually called like that